bio/deeptools/bampefragmentsize/wrapper.py

```
@@ -0,0 +1,54 @@
+__author__ = "Niek Wit"
+__copyright__ = "Copyright 2025, Niek Wit"
+__email__ = "niekwit@gmail.com"
+__license__ = "MIT"
+
+import os
+from snakemake.shell import shell
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+# Get input files
+bam_files = snakemake.input.get("bams", "")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)